lib/addRemove.js

```diff
@@ -64,7 +64,7 @@ function addAsync(version, useNow) {
 			let versionDir = nvsUse.getVersionDir(version);
```

Comment:
[I do have a .editorconfig.](https://github.com/jasongin/nvs/blob/master/.editorconfig) And GitHub _mostly_ respects it, except for the parts of the diff that show unchanged lines.